### PR TITLE
Updtaed the faq.html file

### PR DESCRIPTION
**Pull request by ASHWINIAJIT**
- Added the Status site which uses Next.js, TypeScript
- Added Next.js in the Members site
- Commented out the entries for Main App and Events
- Added the My Site which uses Ember
- Added the Main Site which uses HTML, CSS, Javascript

### CODE REVIEW

File: faq.html [hunk 1]
```diff
@@ -73,12 +73,15 @@ <h3 id="aboutRDS" class="question">How do I contribute to RDS?</h3>
...
-                  <li><a href="https://github.com/Real-Dev-Squad/rds-events-website" target="_blank" rel="noopener noreferrer">Events</a>: React</li>
-                  <li><a href="https://github.com/Real-Dev-Squad/website-crypto" target="_blank" rel="noopener noreferrer">Crypto</a> : Next.js</li>
-                  <li><a href="https://github.com/Real-Dev-Squad/website-emberjs" target="_blank" rel="noopener noreferrer">Main App</a> : Ember</li>
-                  <li><a href="https://github.com/Real-Dev-Squad/website-backend" target="_blank" rel="noopener noreferrer">Backend</a> : Node.js</li>
+                <li><a href="https://github.com/Real-Dev-Squad/website-welcome" target="_blank" rel="noopener noreferrer">Welcome</a> : HTML, CSS, Javascript</li>
+                <li><a href="https://github.com/Real-Dev-Squad/website-www" target="_blank" rel="noopener noreferrer">Main Site</a> : HTML, CSS, Javascript</li>
+                <li><a href="https://github.com/Real-Dev-Squad/website-members" target="_blank" rel="noopener noreferrer">Members</a> : React, Next.js</li>
```
Comment (swarajpure) [inline faq.html:78]: Can we just have `Next.js` for members?

File: faq.html [hunk 2]
```diff
@@ -73,12 +73,15 @@ <h3 id="aboutRDS" class="question">How do I contribute to RDS?</h3>
...
+                <!-- <li><a href="https://github.com/Real-Dev-Squad/rds-events-website" target="_blank" rel="noopener noreferrer">Events</a>: React</li> -->
```
Comment (swarajpure) [inline faq.html:79]: We won't be sending comments in production. Can we please not have them?

File: faq.html [hunk 3]
```diff
@@ -73,12 +73,15 @@ <h3 id="aboutRDS" class="question">How do I contribute to RDS?</h3>
...
+                <li><a href="https://github.com/Real-Dev-Squad/website-crypto" target="_blank" rel="noopener noreferrer">Crypto</a> : Next.js</li>
+                <li><a href="https://github.com/Real-Dev-Squad/website-status" target="_blank" rel="noopener noreferrer">Status</a> : Next.js, TypeScript</li>
+                <!-- <li><a href="https://github.com/Real-Dev-Squad/website-emberjs" target="_blank" rel="noopener noreferrer">Main App</a> : Ember</li> -->
+                <li><a href="https://github.com/Real-Dev-Squad/website-my" target="_blank" rel="noopener noreferrer">My Site</a> : Ember</li>
+                <li><a href="https://github.com/Real-Dev-Squad/website-backend" target="_blank" rel="noopener noreferrer">Backend</a> : Node.js</li>
```
Comment (swarajpure) [inline faq.html:84]: I like how you have maintained the order of the projects... From simple HTML, CSS to frontend frameworks and then backend 😁

**Review by swarajpure — CHANGES_REQUESTED** [review]
Good job there!

Congratulations on your first PR in Real Dev Squad! 🎉 

Suggesting some minor changes

**Review by swarajpure — APPROVED** [review]
Great. Thank you! 🎉

**Review by Rucha1499 — APPROVED** [review]
Good job! Thank you 🎉

**Pull request merged**